content/300-guides/200-deployment/201-serverless/150-deploy-to-azure-functions.mdx

```diff
@@ -1,640 +1,42 @@
 ---
 title: 'Deploy to Azure Functions'
-metaTitle: 'How to deploy a REST API to Azure Functions with Azure SQL'
+metaTitle: 'How to deploy an Azure Function using Prisma'
 metaDescription: 'Learn how to deploy a Prisma based REST API to Azure Functions and connect to an Azure SQL database'
 ---
 
 <TopBlock>
 
-This guide explains how to set up and deploy a Prisma based Node.js REST API to [Azure Functions](https://azure.microsoft.com/en-us/services/functions/) with [Azure SQL](https://azure.microsoft.com/en-us/services/sql-database/) as the database.
+This guide explains how to avoid common issues when deploying a Node.js-based function app to Azure using [Azure Functions](https://azure.microsoft.com/en-us/services/functions/).
 
-Azure Functions is a serverless deployment platform. You do not need to maintain infrastructure to deploy your code. Azure SQL Database is a relational database service built for the cloud with automatic scaling.
-
-In this guide, you create the necessary resources in Azure, create the database schema with Prisma Migrate. You then deploy a Node.js REST API with resource endpoints that use Prisma Client to handle database operations against the Azure SQL database.
-
-This guide shows how you can use Prisma in the Azure cloud, focusing on Azure Functions and Azure SQL. The starting point is the [Prisma Azure Functions example](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/azure-functions). This is a REST API for a simple blog with two models: `User` and `Post` (_one-to-many_). The example contains REST endpoints preconfigured as serverless functions. It uses Prisma Client to fetch, create, and delete records from a database.
+Azure Functions is a serverless deployment platform. You do not need to maintain infrastructure to deploy your code.
 
 With Azure Functions, the fundamental building block is the [function app](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference#function-app). A function app provides an execution context in Azure in which your functions run. It is comprised of one or more individual functions that Azure manages, deploys, and scales together. You can organize and collectively manage multiple functions as a single logical unit.
```

Comment:
maybe combine this paragraph with the above sentence? no strong opinions either way.